core/dispatcher/src/main/scala/whisk/core/container/Container.scala

```diff
@@ -53,32 +53,28 @@ class Container(
 
     def details: String = {
         val name = containerName getOrElse "??"
-        val id = containerId getOrElse "??"
+        val id = containerId.id
         val ip = containerHostAndPort getOrElse "??"
         s"container [$name] [$id] [$ip]"
     }
 
-    def pause(): Unit = {
-        containerId map pauseContainer
-    }
+    def pause(): Unit = pauseContainer(containerId)
 
-    def unpause(): Unit = {
-        containerId map unpauseContainer
-    }
+    def unpause(): Unit = unpauseContainer(containerId)
 
     /**
      * A prefix of the container id known to be displayed by docker ps.
      */
     lazy val containerIdPrefix: String = {
         // docker ps contains only a prefix of the id
-        containerId map { _.take(8) } getOrElse "anon"
+        containerId.id.take(8)
     }
 
     /**
      * Gets logs for container.
      */
     def getLogs()(implicit transid: TransactionId): String = {
-        getContainerLogs(containerId) getOrElse ""
```

Comment:
It looks like this access to logs is used only in tests (checking that logs contain specific `String`s), where always getting a (possibly empty) `String` is a reasonable interface.